labs/iot/gps-lab/steps/set-up-web-app.md

```diff
@@ -0,0 +1,25 @@
+# Run the Web Appliction
+
+In this step you will run the web application.
+
+## Steps
+
+1. Open the server solution in Visual Studio or Visual Studio code
+2. Install Flask and SocketIO
+
+    ```sh
+    pip install Flask
+    ```
+
+     ```sh
```

Comment:
Will this install using Python 2 or 3? What about folks on Mac or Linux for 2 vs 3? Would `pip3` be better?

Any reason for not using virtual environments and a requirements.txt file?

Reply:
put it under requirements.txt and updated instructions